duckduckgo/macos-browser
# Refresh the Autofill panel after importing

Task/Issue URL: https://app.asana.com/0/1199230911884351/1202060497997056/f
Tech Design URL:
CC:

**Description**:

This PR refreshes the Autofill panel after importing data.

**Steps to test this PR**:
1. Clear out all of your Autofill data using the Debug menu -> Reset Secure Vault Data
1. Open the Logins+/Autofill panel and use the empty state Import button to import data
1. Verify that the panel shows your data after importing

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
